test/run_speedy_with_output.jl

```diff
@@ -23,12 +23,12 @@ end
 
 @testset "Restart from output file" begin
     
-    p = run_speedy(Float32, initial_conditions=:restart, restart_id=1)
+    p = run_speedy(Float32, initial_conditions=StartFromFile, restart_id=1)
```

Comment:
This line is redundant anyway, we can just delete it.

Reply:
done